app/Notifications/NewRequestFillReject.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
+        if ($notifiable->notification?->block_notifications == 1) {
```

Comment:
Should add the check to not notify the filler if they're the rejecter

Reply:
Added self check.